constants.py

```diff
@@ -50,10 +50,14 @@
 builders_galera_mtr = [
     "aarch64-debian-12",
     "amd64-fedora-39",
+    "amd64-fedora-40",
+    "amd64-ubuntu-2004-debug",
```

Comment:
2004 it's a typo or do we want to add Galera tests to a Protected branches builder?

Reply:
typo. Thanks. Protected branch builders should be a bit more planned.